### PR TITLE
Updated gatekeeper to 3.10

### PR DESCRIPTION
**Pull request by garethahealy**
$subject

cc: @redhat-cop/rego-policies

### CODE REVIEW

**Comment by garethahealy** [comment]
@pabrahamsson can you review/merge when you've got 5

File: _test/deploy-gatekeeper.sh [hunk 1]
```diff
@@ -53,6 +53,9 @@ deploy_gatekeeper() {
   oc create clusterrole gatekeeper-allow-anyuid-scc --verb=use --resource=securitycontextconstraints.security.openshift.io --resource-name=anyuid
   oc create rolebinding gatekeeper-anyuid-scc --serviceaccount=gatekeeper-system:gatekeeper-admin --clusterrole=gatekeeper-allow-anyuid-scc -n gatekeeper-system
 
+  oc patch Deployment/gatekeeper-audit --type json -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/securityContext/runAsUser"}]' -n gatekeeper-system
+  oc patch Deployment/gatekeeper-controller-manager --type json -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/securityContext/runAsUser"}]' -n gatekeeper-system
+
```
Comment (pabrahamsson) [inline _test/deploy-gatekeeper.sh:58]: Not a blocker and happy to merge as is, but lowercase would make it consistent.
```suggestion
  oc patch deployment/gatekeeper-audit --type json -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/securityContext/runAsUser"}]' -n gatekeeper-system
  oc patch deployment/gatekeeper-controller-manager --type json -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/securityContext/runAsUser"}]' -n gatekeeper-system

```

**Review by pabrahamsson — COMMENTED** [review]
Looks good @garethahealy, just a comment/question inline.

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**